Tribler/Main/vwxGUI/home.py

```diff
@@ -1059,8 +1059,14 @@ def SetData(self, delayedResult):
         for torrent in torrents:
             data.append((torrent.infohash, [torrent.name], torrent, ThumbnailListItemNoTorrent))
 
-        self.list.SetData(data)
-        self.list.SetupScrolling()
+        # This may throw a PyDeadObject error when a test is finishing
+        # but a delayedtask (GuiDBHandler) calls SetData with delayedResult.get().
```

Comment:
Yeah, we have pile of those. Simply because there where so many and we had no time to investigate it.

Any work around will have to be undone once we split libtribler from the gui, so its just pushing stuff back in time.

If you find how this is being scheduled, maybe we can fix the source of the problem in a more permantent way.